src/PythonMigrationViewExtension/Controls/MigrationAssistantWarning.xaml.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Windows;
+using System.Windows.Controls;
+using Dynamo.PythonMigration.MigrationAssistant;
+
+namespace Dynamo.PythonMigration.Controls
+{
+    /// <summary>
+    /// Interaction logic for MigrationAssistantWarning.xaml
+    /// </summary>
+    internal partial class MigrationAssistantWarning : Window
```

Comment:
why use partial, are you defining the codebehind somewhere else as well?

Reply:
i guess i dont really have a good answer for this, but by default all codebehind files are partial classes.

Reply:
fair enough, I see that most Dynamo controls have the same default left there, so seems okay to stick with it.